keysManagement/errors.go

```diff
@@ -6,3 +6,4 @@ var errDuplicatedKey = errors.New("duplicated key found")
 var errMissingPublicKeyDefinition = errors.New("missing public key definition")
 var errNilKeyGenerator = errors.New("nil key generator")
 var errNilP2PIdentityGenerator = errors.New("nil p2p identity generator")
+var errInvalidValue = errors.New("invalid value")
```

Comment:
could have grouped them in a `var (  )` group

Reply:
done